src/widget/wnumberrate.cpp

```diff
@@ -32,11 +31,14 @@ void WNumberRate::setup(const QDomNode& node, const SkinContext& context) {
 void WNumberRate::setValue(double dValue) {
     double vsign = dValue - 1;
 
-    char sign = '+';
-    if (vsign < -0.00000001) {
+    QString sign(' ');
+    if(vsign > 0) {
+        sign = '+';
+    }
+    if(vsign < 0) {
```

Comment:
If the number is -0.0000000001 with will still display as -0.00 I think. I suggest to check if it's greater or equal 0.0001 or less or equal -0.0001.